docs/src/pages/[platform]/theming/authenticator-theming.mdx

```diff
@@ -0,0 +1,89 @@
+import { Fragment } from '@/components/Fragment';
+import { Example, ExampleCode } from '@/components/Example';
+import { Authenticator } from '@aws-amplify/ui-react';
+import Link from 'next/link';
+import {
+  AuthenticatorButtonClassExample,
+  AuthenticatorClasses,
+} from './examples';
+
+A Theme is a structured collection of design decisions that change the appearance of a UI library. An Amplify UI theme allows application wide visual changes through the use of targeted CSS classes and variables.
+
+## Getting started
+
+<Fragment>
+  {({ platform }) =>
+    import(`../components/authenticator/quick-start-example.${platform}.mdx`)
+  }
+</Fragment>
```

Comment:
This snippet is fairly specific to Authenticator setup instead of theme, and is duplicate of https://ui.docs.amplify.aws/angular/components/authenticator#quick-start. 

Do you think we can remove this section? (Or maybe an alert linking to `authenticator#quick-start` link). Open to thoughts!